CRCTray.cs

```diff
@@ -228,39 +229,28 @@ async private void StopMenu_Click(object sender, EventArgs e)
         async private void StartMenu_Click(object sender, EventArgs e)
         {
             // Check using get-config if pullSecret is configured
-            var configs = await TaskHelpers.TryTaskAndNotify(TaskHandlers.ConfigView,
+            var pullsecret = await TaskHelpers.TryTaskAndNotify(TaskHandlers.GetPullSecret,
                 String.Empty,
                 String.Empty,
                 String.Empty);
 
-            if(configs == null)
+            if (!pullsecret)

```

Comment:
This check is not working as expected, the pull-secret prompt always appears when you click on start even when its already saved in the credential store

Reply:
If the pull secret is not stored the api returns a 404 http status not found

Reply:
I'll have to check this, but this always returns `false`? The idea is that an exception returns a `default` typed result. I see that the 200, does not return any value, so this means the `bool` type will therefore have no value.

```
return getResultsForBasicCommand<bool>(BasicCommands.PullSecret);
```

Reply:
AggregateException was thrown instead, containing the APINotFound